src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -205,13 +199,13 @@ def body(self):
         self.cmd('network express-route circuit list --resource-group {}'.format(self.resource_group),
             checks=NoneCheck())
 
-class NetworkExpressRouteCircuitScenarioTest(VCRTestBase):
+class NetworkExpressRouteCircuitScenarioTest(ResourceGroupVCRTestBase):
```

Comment:
>ResourceGroupVCRTestBase [](start = 45, length = 24)

no need to convert this one yet since it doesn't actually use the resource group.  This will help us track which tests need to be updated